manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Yourkit.JavaProfiler
+PackageVersion: 2022.3
+PackageLocale: en-US
+Publisher: YourKit GmbH
+PackageName: Java Profiler

```

```suggestion
PackageName: YourKit Java Profiler
```
Should match Control Panel